woocommerce/woocommerce
# Add sort handle column when adding new tags.

### All Submissions:






### Changes proposed in this Pull Request:



Closes #19855 .

### How to test the changes in this Pull Request:

1. WP admin > Products > Attributes > Create new attribute.
2. Configure terms, add terms.
3. Observe no sort handle.
4. Refresh page -- handles are displayed.
5. Click on Quick Edit and update term.
6. Observe missing sort handle.
7. Apply branch.
8. WP admin > Products > Attributes > Configure terms.
9. Add term.
10. Sort handle appears.
11. Quick edit term.
12. Sort handle re-appears.




### Other information:




### Changelog entry

> Fixed sort handle when adding new terms for Products.
